CMakeLists.txt

```diff
@@ -367,11 +377,17 @@ endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 
 # enables testing and creates Make check command
 enable_testing()
-set(CMAKE_CTEST_COMMAND ctest -V)
+set(CMAKE_CTEST_COMMAND ctest)
 add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND})
 
 add_subdirectory(src)
 
+# Now create a useable config.h
+configure_file(
```

Comment:
and also move the line of https://github.com/ceph/ceph/pull/9156/files#diff-af3b638bc2a3e6c650974192a53c7291R362 there. so it would be easier for future maintainer to understand our intention.